cdk/lib/clamav-scanner-stack.ts

```diff
@@ -47,13 +48,18 @@ export class ClamavScannerStack extends Stack {
       },
     });
 
-    // NOTE: ckan storage path will be in: /mnt/ytp_files/ckan
     clamavContainer.addMountPoints({
       containerPath: '/var/lib/clamav',
       readOnly: false,
       sourceVolume: 'clamav_files',
     });
 
+    const clamavSecurityGroup = new SecurityGroup(this, 'clamav-security-group', {
+      vpc: props.cluster.vpc,
+      description: "clamav container sercurity group"
+    });
+    clamavSecurityGroup.connections.allowFrom(props.clamavFileSystem, Port.tcp(2049), 'EFS connection (clamav)')
```

Comment:
```suggestion
   ```

Not required, should be removed from elsewhere as well, but its not the problem of this PR.